### PR TITLE
feat: Create 2022-03-25-community-call

### PR DESCRIPTION
**Pull request by Francescopog**
@bfabio titolo da definire

## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Create 2022-03-25-community-call

@bfabio titolo da definire
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 3b1740e638beda20ce2114a065135ab47637fa10
</p>

File: _posts/it/2022-03-25-community-call-sinergia-universita-pa.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Community call - Design: la sinergia possibile tra Università e Pubbliche amministrazioni"
+subtitle: "Il 6 aprile scopri come le Università possono contribuire alla trasformazione dei servizi pubblici digitali"
+date: 2022-03-25 10:00:00 +0200
+tags: developers designers università
```
Comment (bfabio) [inline _posts/it/2022-03-25-community-call-sinergia-universita-pa.md:6]:
```suggestion
tags: developers designers universita
```

Comment (bfabio) [reply _posts/it/2022-03-25-community-call-sinergia-universita-pa.md:6]: Purtroppo i tag non supportano le accentate

File: _posts/it/2022-03-25-community-call-sinergia-universita-pa.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Community call - Design: la sinergia possibile tra Università e Pubbliche amministrazioni"
```
Comment (bfabio) [inline _posts/it/2022-03-25-community-call-sinergia-universita-pa.md:3]:
```suggestion
title: "Community call: la sinergia possibile tra Università e Pubbliche amministrazioni"
```

File: _posts/it/2022-03-25-community-call-sinergia-universita-pa.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Consapevole del valore generato sia per le amministrazioni pubbliche sia per gli studenti della disciplina, Designers Italia intende così invitare **professori, ricercatori, tesisti, studenti a entrare a far parte della community** e ad avere un ruolo attivo nella disseminazione della cultura progettuale nel settore pubblico.
+
+Appuntamento dunque alla **community call** di mercoledì **6 aprile**, dove torneremo tra i banchi per parlare dell’Atlante della trasformazione digitale della PA e dare visibilità ad altri **esempi virtuosi di collaborazione tra Università ed enti pubblici** nell’ambito della progettazione per la Pubblica Amministrazione, con l’obiettivo di raccontare le iniziative intraprese, ispirare interlocutori e, secondo una logica aperta, ampliare la conoscenza e le risorse sul tema.
+
+**[Registrati all’evento per partecipare](https://mobilizon.it/events/a6349415-b43f-4b63-8c92-c06a647a1cc0)**. Ti aspettiamo il 6 aprile alle 15.30!
```
Comment (bfabio) [inline _posts/it/2022-03-25-community-call-sinergia-universita-pa.md:21]:
```suggestion
**[Registrati all’evento per partecipare](https://mobilizon.it/events/a6349415-b43f-4b63-8c92-c06a647a1cc0)**. Ti aspettiamo il **6 aprile alle 15.30**!
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**